src/Lykke.Service.FixGateway.Services/NewOrderRequestHandler.cs

```diff
@@ -147,49 +202,31 @@ public void Handle(NewOrderSingle request)
             };
         }
 
-        private static Guid GenerateOrderId()
+        private async Task<MatchingEngine.Connector.Abstractions.Models.LimitOrderFeeModel> GetLimitOrderFeeAsync(string assetPairId, OrderAction orderAction)
         {
-            return Guid.NewGuid();
-        }
-
-        private async Task<bool> ValidateRequestAsync(Guid newOrderId, NewOrderSingle request)
-        {
-            var clOrdId = request.ClOrdID.Obj;
-            var orderExists = await _clientOrderIdProvider.CheckExistsAsync(clOrdId);
-            Message reject = null;
-            if (orderExists)
-            {
-                reject = CreateRejectResponse(newOrderId, request, OrdRejReason.DUPLICATE_ORDER);
-            }
-
-            var allSymbols = (await _assetsServiceWithCache.GetAllAssetPairsAsync(_tokenSource.Token)).Select(a => a.Id).ToHashSet();
+            var assetPair = await _assetsServiceWithCache.TryGetAssetPairAsync(assetPairId);
+            var fee = await _feeCalculatorClient.GetLimitOrderFees(_credentials.ClientId.ToString("D"), assetPairId, assetPair?.BaseAssetId, _mapper.Map<FeeCalculator.AutorestClient.Models.OrderAction>(orderAction));
```

Comment:
>ClientId.ToString("D")
Why using explicit format?

Reply:
To consistency. I saw in many places that client id compared as a string. I would like to be sure that the string is equal everywhere.